### DISCUSSION TITLE
Folder structure

### DISCUSSION THREAD

**Discussion by jsmariegaard** [DHI/python-package-development#15] [General]
│   setup.py
	│
	├───notebooks
	│       notebook_A.ipynb
	│
	├───scripts
	│       main.py
	│
	├───tests
	│   │   test_cleaning.py
	│   │   __init__.py
	│   │
	│   └───testdata
	│           example_data1.csv
	│
	└───tscleaner
			cleaning.py
			plotting.py
			__init__.py

**Comment by buchmann-dhi** [comment]
@jsmariegaard How do you fix the path when moving scripts into notebook/ and scripts/, ie how do the notebook and script find the base package?

**Reply by jsmariegaard** [reply]
Hi @buchmann-dhi, I use relative paths for these, e.g. in notebooks I refer to:

    fn = "../tests/testdata/example_data1.csv"
